code_fellows/stack.py

```diff
@@ -9,12 +9,10 @@ def __init__(self, values=None):
 
     def push(self, value):
         self.linked_list.push(value)
+        self.head = self.linked_list.head
```

Comment:
Unclear why this addition?  Even if a stack did have a head, this wouldn't be updated on other actions effecting the linked list.  `self.head` doesn't point to `self.linked_list.head` because it is also a pointer to a Node.  They both would start out pointing to the same Node, but as soon as the ll.head pointer targets a different node they will no longer be the same.  Stacks don't need heads anyway.